### ISSUE TITLE
1.7.10 Click Turtles

### ISSUE THREAD

**Issue by donkaike** [KnightMiner/KnightPeripherals#4]
I cannot create a click turtle in 1.7.10.
I Have a claw, but I cannot put it on turtle!

**Comment by KnightMiner** [comment]
Is the recipe for the claw enabled in the config?

**Comment by donkaike** [comment]
Yes! I make my claw with craft. But I cannot craft it with the turtle or put it in turtle with right click.
